groundstation/backend_api/validators.py

```diff
@@ -80,6 +80,8 @@ class UserValidator(Schema):
     """
     username = fields.String(required=True)
     password = fields.String(required=True)
+    is_admin = fields.Boolean(required=True)
+    creating_admin_user = fields.String(required=True)
```

Comment:
```suggestion
    is_admin = fields.Boolean(required=True)
    creating_admin_user = fields.String(required=False)
```
The `creating_admin_user`/`creator` field is not required since we have the field set to 'nullable' in models.py.

The `is_admin` can also be set to not required, or you can change the `fake_user_as_dict()` method here on [Line 86 of groundstation/tests/utils.py](https://github.com/AlbertaSat/ex2_ground_station_website/blob/99ab4e1667d583e24cfebb23d88366a789324736/groundstation/tests/utils.py#L86) to include the `is_admin` field.

These changes should make all the test cases pass (you can run `python3 manage.py test test_api.TestUserService` to run just the user related tests)